superset-frontend/src/dashboard/components/RefreshIntervalModal.tsx

```diff
@@ -104,7 +119,7 @@ class RefreshIntervalModal extends React.PureComponent {
             <FormLabel>{t('Refresh frequency')}</FormLabel>
             <Select
               options={options}
-              value={this.state.refreshFrequency}
+              value={this.state.refreshFrequency as ValueType<any>}
```

Comment:
is `any` the best we can do here? perhaps `string | number` since this is human provided input

Reply:
![image](https://user-images.githubusercontent.com/8277264/102244994-ad4fb300-3f05-11eb-8f2f-3521d34174de.png)